WcaOnRails/app/views/competitions_mailer/results_submitted.erb

```diff
@@ -0,0 +1,3 @@
+<p>
+  <%= md @message %>
```

Comment:
Can you include a helpful message here for the WRT? Something like:

```
Results Team,

<%= @delegate.name %> has submitted the attached results for <%= @competition.name %>. You can upload the results at <%= link_to "https://www.worldcubeassociation.org/results/admin/upload_results.php?competitionId=#{@competition.id}" %>.

Here's a message from <%= @delegate.name %>:

<blockquote><%= @message %></blockquote>
```

Reply:
Oh I forgot this - fixing now

Reply:
Would you be happy with something like this:
```
<p>Results Team,</p>

<p>Please find attached the results for <%= @competition.name %>. You can upload them <a href=<%= "https://www.worldcubeassociation.org/results/admin/upload_results.php?competitionId=#{@competition.id}" %>>here</a>.</p>

<p>Here's a message from the delegate:</p>

<blockquote><%= md @message %></blockquote>
```
?
In order to have access to the sender here, I need to define `@delegate` in the mailer. Since it doesn't know which user is sending the email, it would need to be passed as a parameter from the results submission controller. There are already 4 parameters so it's starting to feel messy already.